.travis.yml

```diff
@@ -69,7 +69,7 @@ jobs:
   include:
   - &ppa-upload
     stage: upload to ppa:mir-team
-    env: RELEASE=18.04
+    env: RELEASE=20.04
```

Comment:
You want to replace `20.04` with `20.10` below?

Reply:
Done